.github/workflows/release.yml

```diff
@@ -55,3 +55,15 @@ jobs:
         if: >- # "create" workflows run separately from "push" & "pull_request"
           github.event_name == 'release'
         uses: pypa/gh-action-pypi-publish@release/v1
+
+  forum_post:
+    if: "${{ github.event_name == 'release' }}"
+    needs: release
+    runs-on: ubuntu-24.04
+
+    steps:
+      - name: Retrieve the forum post script from team-devtools
+        run: curl -O https://raw.githubusercontent.com/ansible/team-devtools/main/.github/workflows/forum_post.py
```

Comment:
Don't use curl for these. Just rely on a reusable action to do the posting. Basically we should be able to get the posting for free in other repos once we implment it in release workflow from team-devtools.